css/card.css

```diff
@@ -486,7 +478,7 @@ section#gacha-controls input {
   background-blend-mode: overlay;
   z-index: 2;
   filter: brightness(1) contrast(1);
-  transition: all 0.33s ease;
+  transition: filter 0.33s ease;
```

Comment:
explicit definition to reduce browser load on possibly transitioning all css properties associated with this element